spanner/tests/system/utils/populate_streaming.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Populate spanner databases with data for streaming system tests."""
+
+import os
+
+from google.cloud.spanner import Client
+from google.cloud.spanner.keyset import KeySet
+from google.cloud.spanner.pool import BurstyPool
+
+
+INSTANCE_NAME = 'gcp-streaming-systests'
+DATABASE_NAME = 'testing'
+
+DDL = """\
+CREATE TABLE four_kay (
+    pkey INT64,
+    chunk_me STRING(4096) )
+    PRIMARY KEY (pkey);
+CREATE TABLE forty_kay (
+    pkey INT64,
+    chunk_me STRING(40960) )
+    PRIMARY KEY (pkey);
+CREATE TABLE four_hundred_kay (
+    pkey INT64,
+    chunk_me STRING(409600) )
+    PRIMARY KEY (pkey);
+CREATE TABLE four_meg (
+    pkey INT64,
+    chunk_me STRING(2097152),
+    chunk_me_2 STRING(2097152) )
+    PRIMARY KEY (pkey);
+"""
+
+DDL_STATEMENTS = [stmt.strip() for stmt in DDL.split(';') if stmt.strip()]
+
+
+def print_func(message):
```

Comment:
See above:  the `tests/` tree is not importable when running a script.  I have changes which jam it into `sys.path` in my local checkout, if that seems best.